spec.bs

```diff
@@ -316,11 +316,28 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
 
 To <dfn>check interest group permissions</dfn> given an [=origin=]
 |ownerOrigin|, an [=origin=] |frameOrigin|, and a [=boolean=] |isJoin|:
-1. If |ownerOrigin| is [=same origin=] to |frameOrigin|, then return true.
-1. Let |permissionsUrl| be the result of [=building an interest group permissions url=]
-  with |ownerOrigin| and |frameOrigin|.
-1. Let |request| be the result of [=creating a request=] with |permissionsUrl|,
-  "`application/json`", and null.
+1. If |ownerOrigin| is [=same origin=] with |frameOrigin|, then return true.
+1. Let |permissionsUrl| be the result of [=building an interest group permissions url=] with
+  |ownerOrigin| and |frameOrigin|.
+1. Let |request| be a new [=request=] with the following properties:
+  :   [=request/URL=]
+  ::  |permissionsUrl|
+  :   [=request/header list=]
+  ::  «`Accept`: `application/json`»
+  :   [=request/client=]
+  ::  `null`

```

Comment:
I believe a null client means you have to pass [useParallelQueue](https://fetch.spec.whatwg.org/#fetch-useparallelqueue) to Fetch and then expect the callbacks to run in parallel. https://github.com/whatwg/fetch/issues/1673 might clarify this, but since all of these requests appear to happen in parallel anyway, it's probably best to set useParallelQueue to true in each call to Fetch.

Reply:
+1!